File: WaterManager/src/main/java/pissir/watermanager/model/user/GestoreAzienda.java
package pissir.watermanager.model.user;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import pissir.watermanager.model.item.Azienda;

/**
 * @author Almasio Luca
 * @author Borova Dritan
 * @author Gattico Alessandro
 */
@Setter
@Getter
@NoArgsConstructor
public class GestoreAzienda extends UserProfile {
	
	private Azienda azienda;
	
	
	public GestoreAzienda(int id, String nome, String cognome, String username, String mail, String password,
						  boolean enabled) {
		super(id, nome, cognome, username, mail, password, UserRole.GESTOREAZIENDA, enabled);
		this.azienda = null;
	}
	
}

File: WaterManagerUI/Model/User/GestoreAzienda.cs
using WaterManagerUI.Model.Item;

namespace UserInterfaceWaterManager.Model.User
{
    public class GestoreAzienda : UserProfile
    {
        public Azienda azienda { get; set; }


        public GestoreAzienda(int id, String nome, String cognome, String username, String mail, String password,
            Azienda azienda) : base(id, nome, cognome, username, mail, password, UserRole.GESTOREAZIENDA)
        {
            this.azienda = azienda;
        }
    }
}